test/specs/commands/update_forging_status.js

```diff
@@ -0,0 +1,197 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { setUpCommandUpdateForgingStatus } from '../../steps/setup';
+import * as given from '../../steps/1_given';
+import * as when from '../../steps/2_when';
+import * as then from '../../steps/3_then';
+
+describe('update forging status command', () => {
+	beforeEach(setUpCommandUpdateForgingStatus);
+	Given('a Lisk API instance', given.aLiskAPIInstance, () => {
+		Given(
+			'a Vorpal instance with a UI and an active command that can prompt',
+			given.aVorpalInstanceWithAUIAndAnActiveCommandThatCanPrompt,
+			() => {
+				Given('an action "update forging status"', given.anAction, () => {
+					Given(
+						'an options object with a password set to "randompassword"',
+						given.anOptionsObjectWithPasswordSetTo,
+						() => {
+							Given(
+								'the password can be retrived from its source',
+								given.thePasswordCanBeRetrievedFromItsSource,
+								() => {
+									Given('a status "enable"', given.aStatus, () => {
+										Given(
+											'a public key "1b5a93c7622c666b0228236a70ee1a31407828b71bfb6daaa29a1509e87d4d3c"',
```

Comment:
This step is repeated several times, which is a sign it should be further up in the hierarchy.